.github/actions/install/action.yml

```
@@ -0,0 +1,15 @@
+name: Setup python environenment
```

```suggestion
name: Setup python environment
```
Could you also name the directory so it matches?

---

:+1: